blacksauce/src/styles/_forms.scss

```diff
@@ -0,0 +1,48 @@
+.para--leading {
+    line-height: 150%;
+}
+
+.footerPadding {
+    padding: 5%;
+}
+
+label {
+    color: white;
+    display: block;
+    font-family: "Open Sans", sans-serif;
+    padding-bottom: 1em;
+}
+
+input[type="button"],
+input[type="submit"],
+input[type="reset"] {
+    background-color: #f28705;
```

Comment:
Have you considered using a variable for this brand color? It's referenced at least 7 times already in your stylesheets.